### PR TITLE
Alert user about checksum addreess

### PR DESCRIPTION
**Pull request by imstar15**
**Copied from MetaMask**

image

image

**Lowercase**

image

**Empty record**

**Testnet has no record, but Mainnet has.**

image

**Testnet and Mainnet has no record.**

image

### CODE REVIEW

File: src/common/translations/pt-BR.json [hunk 1]
```diff
@@ -609,6 +609,26 @@
     "feeCalculationError": {
       "title": "Erro de cálculo de taxa",
       "body": "Ocorreu um erro ao calcular o custo. Por favor, tente novamente."
+    },
+    "invalidRskAddress": {
+      "title": "Endereço RSK inválido",
+      "body": {
+        "normal": "O endereço do destinatário inserido é um endereço RSK inválido. Certifique-se de que o endereço do destinatário é um endereço RSK válido e verifique-o no %{explorerName} antes de enviar.",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:616]:
```suggestion
        "normal": "O endereço inserido é um endereço RSK inválido. Certifique-se de que o endereço do destinatário é um endereço RSK válido e verifique-o no %{explorerName} antes de enviar.",
```

File: src/common/translations/pt-BR.json [hunk 2]
```diff
@@ -609,6 +609,26 @@
...
+        "ethereum": "O endereço do destinatário inserido parece ser um endereço Ethereum. Este é um bug conhecido para valores copiados do Metamask. Certifique-se de que o endereço do destinatário é um endereço RSK válido e verifique-o no %{explorerName} antes de enviar."
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:617]:
```suggestion
        "ethereum": "O endereço inserido parece ser um endereço Ethereum. Este é um problema conhecido e ocorre quando se copia endereços da Metamask. Certifique-se de que o endereço do destinatário é um endereço RSK válido e verifique-o no %{explorerName} antes de enviar."
```

File: src/common/translations/pt-BR.json [hunk 3]
```diff
@@ -609,6 +609,26 @@
...
+      },
+      "question": "Por que você está vendo esta mensagem?",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:619]:
```suggestion
      "question": "Por que estou vendo esta mensagem?",
```

File: src/common/translations/pt-BR.json [hunk 4]
```diff
@@ -609,6 +609,26 @@
...
+      "confirm": "Ok, envie assim mesmo",
+      "cancel": "Volte e entre novamente",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:621]:
```suggestion
      "cancel": "Voltar e reinserir",
```

File: src/common/translations/pt-BR.json [hunk 5]
```diff
@@ -609,6 +609,26 @@
...
       "body": "Ocorreu um erro ao calcular o custo. Por favor, tente novamente."
+    },
+    "invalidRskAddress": {
+      "title": "Endereço RSK inválido",
+      "body": {
+        "normal": "O endereço do destinatário inserido é um endereço RSK inválido. Certifique-se de que o endereço do destinatário é um endereço RSK válido e verifique-o no %{explorerName} antes de enviar.",
+        "ethereum": "O endereço do destinatário inserido parece ser um endereço Ethereum. Este é um bug conhecido para valores copiados do Metamask. Certifique-se de que o endereço do destinatário é um endereço RSK válido e verifique-o no %{explorerName} antes de enviar."
+      },
+      "question": "Por que você está vendo esta mensagem?",
+      "confirm": "Ok, envie assim mesmo",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:620]:
```suggestion
      "confirm": "Ok, enviar assim mesmo",
```

File: src/common/translations/pt-BR.json [hunk 6]
```diff
@@ -609,6 +609,26 @@
...
+      "cancel": "Volte e entre novamente",
+      "asset": "De ativos",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:622]:
```suggestion
      "asset": "Ativo",
```

File: src/common/translations/pt-BR.json [hunk 7]
```diff
@@ -609,6 +609,26 @@
...
+      "address": "Endereço",
+      "balance": "Saldo"
+    },
+    "emptyRecordAddress": {
+      "title": "Endereço de destinatário de registro vazio",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:627]:
```suggestion
      "title": "Endereço sem registro",
```

File: src/common/translations/pt-BR.json [hunk 8]
```diff
@@ -609,6 +609,26 @@
...
+      "body": {
+        "normal": "O endereço do destinatário inserido não tem registro em %{explorerName} e tem %{balance} %{otherSymbolName} na rede RSK ooo. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço em RSK %{type}?",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:629]:
```suggestion
        "normal": "O endereço inserido não tem registro na %{explorerName} e tem %{balance} %{otherSymbolName} na rede RSK. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço RSK da %{type}?",
```

File: src/common/translations/pt-BR.json [hunk 9]
```diff
@@ -609,6 +609,26 @@
...
+        "both": "O endereço do destinatário inserido não tem registro em %{explorerName}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço em RSK %{type}?"
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:630]:
```suggestion
        "both": "O endereço inserido não tem registro na %{explorerName}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço RSK na %{type}?"
```

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Changes Portuguese Brazilian

**Review by chrisli30 — CHANGES_REQUESTED** [review]
Need to update the color of the checkbox to the same color of "The entered recipient address" paragraph

**Review by chrisli30 — APPROVED** [review]

**Pull request merged**